### ISSUE TITLE
[BRO] Urza, Lord Protector and The Mightstone and Weakstone issues

### ISSUE THREAD

**Issue by copperfield42** [magefree/mage#9626]
testing the cards [[Urza, Lord Protector]] and [[The Mightstone and Weakstone]] and their melded form [[Urza, Planeswalker]] in a Constructed freeform two player duel vs the AI I found the following issues:

The Mightstone and Weakstone:
- only draw you 1 card instead of 2
- the mana from it can't be used to cast artifact or to paid for abilities (like the meld from Urza)

Urza, Planeswalker
- the cost reduction from Urza, Lord Protector still apply 
- its +2 only reduce cost by 1 instead of 2
- its +1 does nothing, and its text is wrong, it said that target a player
- its +0 create only 1 soldier token instead of 2 and the text is wrong too

everything else seem to be ok

**Comment by theelk801** [comment]
sorry, I meant to comment here that my commit didn't fix everything

**Comment by awjackson** [comment]
Yeah, it looks like some abilities of the separate cards are still active after merging (this affects both Mishra and Urza--Mishra's front-side triggered ability triggers in a very glitchy fashion when you attack with the back side)

I guess this didn't affect any of the EMN cards because none of them had static abilities and none of them had triggered abilities that could possibly trigger while they were merged.

**Comment by theelk801** [comment]
yup, meld cards are gonna need work

**Comment by copperfield42** [comment]
I found another issue with Urza, Planeswalker: it doesn't a third use with [[The Chain Veil]]

**Comment by theelk801** [comment]
Now that almost all of these problems have been fixed I've spun off the remaining one into #9642
